test/database.ts

```diff
@@ -50,6 +50,7 @@ const fakePfy = extend({}, pfy, {
     promisified = true;
     assert.deepStrictEqual(options.exclude, [
       'batchTransaction',
+      'batchWrite',
```

Comment:
Unit tests should be added for batchwrite

Reply:
Done